app/assets/javascripts/utils/api.js

```diff
@@ -675,6 +675,24 @@ slide_id=${opts.slide_id}`,
     );
   },
 
+  supplement(data) {
```

Comment:
Instead of tossing another method into api.js, this can be put into an onboarding_actions file.